operator/config/crd/bases/core.paladin.io_paladins.yaml

```diff
@@ -68,10 +68,68 @@ spec:
                 required:
                 - authMethod
                 type: object
+              baseLedgerEndpoint:
+                description: BaseLedgerEndpoint specifies the base endpoint for the
+                  ledger
+                properties:
+                  endpoint:
+                    description: Network specifies the configuration when the type
+                      is 'network'.
+                    properties:
+                      authConfig:
+                        properties:
+                          authMethod:
```

Comment:
should this just be `method` given that it is already a child of a field named `authConfig` ( or `auth`)